src/miral/minimal_window_manager.cpp

```diff
@@ -14,13 +14,19 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <algorithm>
 #include <miral/minimal_window_manager.h>
 #include <miral/toolkit_event.h>
 #include <miral/application_info.h>
 #include "application_selector.h"
+#include "mir/geometry/forward.h"
+
 #include <linux/input.h>
 #include <gmpxx.h>
 
+#include <ranges>
+#include <unordered_set>
```

Comment:
We tend to local headers (with `"`), Mir headers (with `<`), system headers (with `<`) and c++ headers (with `<`).

Also, it seems unlikely to need `mir/geometry/forward.h` in an implementation file.

Vis:

```suggestion
#include "application_selector.h"

#include <miral/minimal_window_manager.h>
#include <miral/toolkit_event.h>
#include <miral/application_info.h>

#include <linux/input.h>
#include <gmpxx.h>

#include <algorithm>
#include <ranges>
#include <unordered_set>
```